From f9e93719a99120e93628bf39bd997164a01afda2 Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Fri, 13 May 2016 15:23:27 +0800
Subject: [PATCH 03/93] armv8: aarch32: Disable CONFIG_FSL_LS_PPA for loading
 32-bit OS

The current U-Boot doesn't support loading 32-bit OS with PPA, so
disable CONFIG_FSL_LS_PPA so far.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 include/configs/ls1043ardb.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/ls1043ardb.h b/include/configs/ls1043ardb.h
index c1abc73..af8b441 100644
--- a/include/configs/ls1043ardb.h
+++ b/include/configs/ls1043ardb.h
@@ -9,7 +9,7 @@
 
 #include "ls1043a_common.h"
 
-#define CONFIG_FSL_LS_PPA
+#undef CONFIG_FSL_LS_PPA
 #if defined(CONFIG_FSL_LS_PPA)
 #define CONFIG_ARMV8_PSCI
 #define CONFIG_SYS_LS_PPA_DRAM_BLOCK_MIN_SIZE		(1UL * 1024 * 1024)
-- 
1.7.9.5

